### YamlMime:ManagedReference
items:
- uid: System.DirectoryServices.Protocols.ErrorResponseCategory
  id: ErrorResponseCategory
  children:
  - System.DirectoryServices.Protocols.ErrorResponseCategory.AuthenticationFailed
  - System.DirectoryServices.Protocols.ErrorResponseCategory.ConnectionClosed
  - System.DirectoryServices.Protocols.ErrorResponseCategory.CouldNotConnect
  - System.DirectoryServices.Protocols.ErrorResponseCategory.GatewayInternalError
  - System.DirectoryServices.Protocols.ErrorResponseCategory.MalformedRequest
  - System.DirectoryServices.Protocols.ErrorResponseCategory.NotAttempted
  - System.DirectoryServices.Protocols.ErrorResponseCategory.Other
  - System.DirectoryServices.Protocols.ErrorResponseCategory.UnresolvableUri
  langs:
  - csharp
  name: ErrorResponseCategory
  nameWithType: ErrorResponseCategory
  fullName: System.DirectoryServices.Protocols.ErrorResponseCategory
  type: Enum
  summary: "ErrorResponseCategory 列挙には、可能な DSML エラー応答が含まれています。"
  syntax:
    content: public enum ErrorResponseCategory
  inheritance:
  - System.Enum
  platform:
  - net462
- uid: System.DirectoryServices.Protocols.ErrorResponseCategory.AuthenticationFailed
  id: AuthenticationFailed
  parent: System.DirectoryServices.Protocols.ErrorResponseCategory
  langs:
  - csharp
  name: AuthenticationFailed
  nameWithType: ErrorResponseCategory.AuthenticationFailed
  fullName: System.DirectoryServices.Protocols.ErrorResponseCategory.AuthenticationFailed
  type: Field
  assemblies:
  - System.DirectoryServices.Protocols
  namespace: System.DirectoryServices.Protocols
  summary: "認証に失敗しました。 値が 5 に等しいです。"
  syntax:
    content: AuthenticationFailed
    return:
      type: System.DirectoryServices.Protocols.ErrorResponseCategory
      description: "追加します。"
  exceptions: []
  platform:
  - net462
- uid: System.DirectoryServices.Protocols.ErrorResponseCategory.ConnectionClosed
  id: ConnectionClosed
  parent: System.DirectoryServices.Protocols.ErrorResponseCategory
  langs:
  - csharp
  name: ConnectionClosed
  nameWithType: ErrorResponseCategory.ConnectionClosed
  fullName: System.DirectoryServices.Protocols.ErrorResponseCategory.ConnectionClosed
  type: Field
  assemblies:
  - System.DirectoryServices.Protocols
  namespace: System.DirectoryServices.Protocols
  summary: "DSML プロバイダーは、サーバーに接続されているが、サーバーが要求に応答せず、接続を切断します。 値が 2 に等しいです。"
  syntax:
    content: ConnectionClosed
    return:
      type: System.DirectoryServices.Protocols.ErrorResponseCategory
      description: "追加します。"
  exceptions: []
  platform:
  - net462
- uid: System.DirectoryServices.Protocols.ErrorResponseCategory.CouldNotConnect
  id: CouldNotConnect
  parent: System.DirectoryServices.Protocols.ErrorResponseCategory
  langs:
  - csharp
  name: CouldNotConnect
  nameWithType: ErrorResponseCategory.CouldNotConnect
  fullName: System.DirectoryServices.Protocols.ErrorResponseCategory.CouldNotConnect
  type: Field
  assemblies:
  - System.DirectoryServices.Protocols
  namespace: System.DirectoryServices.Protocols
  summary: "DSML プロバイダーは、サーバーに接続できませんでした。 値は、1 と同じです。"
  syntax:
    content: CouldNotConnect
    return:
      type: System.DirectoryServices.Protocols.ErrorResponseCategory
      description: "追加します。"
  exceptions: []
  platform:
  - net462
- uid: System.DirectoryServices.Protocols.ErrorResponseCategory.GatewayInternalError
  id: GatewayInternalError
  parent: System.DirectoryServices.Protocols.ErrorResponseCategory
  langs:
  - csharp
  name: GatewayInternalError
  nameWithType: ErrorResponseCategory.GatewayInternalError
  fullName: System.DirectoryServices.Protocols.ErrorResponseCategory.GatewayInternalError
  type: Field
  assemblies:
  - System.DirectoryServices.Protocols
  namespace: System.DirectoryServices.Protocols
  summary: "ゲートウェイの内部エラーがあります。 値は 4 に等しいです。"
  syntax:
    content: GatewayInternalError
    return:
      type: System.DirectoryServices.Protocols.ErrorResponseCategory
      description: "追加します。"
  exceptions: []
  platform:
  - net462
- uid: System.DirectoryServices.Protocols.ErrorResponseCategory.MalformedRequest
  id: MalformedRequest
  parent: System.DirectoryServices.Protocols.ErrorResponseCategory
  langs:
  - csharp
  name: MalformedRequest
  nameWithType: ErrorResponseCategory.MalformedRequest
  fullName: System.DirectoryServices.Protocols.ErrorResponseCategory.MalformedRequest
  type: Field
  assemblies:
  - System.DirectoryServices.Protocols
  namespace: System.DirectoryServices.Protocols
  summary: "クライアントは、各要求の要求 ID を指定できませんでした。 値は 3 と等しいです。"
  syntax:
    content: MalformedRequest
    return:
      type: System.DirectoryServices.Protocols.ErrorResponseCategory
      description: "追加します。"
  exceptions: []
  platform:
  - net462
- uid: System.DirectoryServices.Protocols.ErrorResponseCategory.NotAttempted
  id: NotAttempted
  parent: System.DirectoryServices.Protocols.ErrorResponseCategory
  langs:
  - csharp
  name: NotAttempted
  nameWithType: ErrorResponseCategory.NotAttempted
  fullName: System.DirectoryServices.Protocols.ErrorResponseCategory.NotAttempted
  type: Field
  assemblies:
  - System.DirectoryServices.Protocols
  namespace: System.DirectoryServices.Protocols
  summary: "DSML プロバイダーは、要求の要素を実行する試みませんでした。 値は 0 です。"
  syntax:
    content: NotAttempted
    return:
      type: System.DirectoryServices.Protocols.ErrorResponseCategory
      description: "追加します。"
  exceptions: []
  platform:
  - net462
- uid: System.DirectoryServices.Protocols.ErrorResponseCategory.Other
  id: Other
  parent: System.DirectoryServices.Protocols.ErrorResponseCategory
  langs:
  - csharp
  name: Other
  nameWithType: ErrorResponseCategory.Other
  fullName: System.DirectoryServices.Protocols.ErrorResponseCategory.Other
  type: Field
  assemblies:
  - System.DirectoryServices.Protocols
  namespace: System.DirectoryServices.Protocols
  summary: "不明なエラーが発生しました。 値は 7 です。"
  syntax:
    content: Other
    return:
      type: System.DirectoryServices.Protocols.ErrorResponseCategory
      description: "追加します。"
  exceptions: []
  platform:
  - net462
- uid: System.DirectoryServices.Protocols.ErrorResponseCategory.UnresolvableUri
  id: UnresolvableUri
  parent: System.DirectoryServices.Protocols.ErrorResponseCategory
  langs:
  - csharp
  name: UnresolvableUri
  nameWithType: ErrorResponseCategory.UnresolvableUri
  fullName: System.DirectoryServices.Protocols.ErrorResponseCategory.UnresolvableUri
  type: Field
  assemblies:
  - System.DirectoryServices.Protocols
  namespace: System.DirectoryServices.Protocols
  summary: "クライアント プロバイダーは、サーバーに転送されることができる値への URI を解決するのにはできません。 値は 6 になります。"
  syntax:
    content: UnresolvableUri
    return:
      type: System.DirectoryServices.Protocols.ErrorResponseCategory
      description: "追加します。"
  exceptions: []
  platform:
  - net462
references:
- uid: System.Enum
  isExternal: false
  name: System.Enum
- uid: System.DirectoryServices.Protocols.ErrorResponseCategory.AuthenticationFailed
  parent: System.DirectoryServices.Protocols.ErrorResponseCategory
  isExternal: false
  name: AuthenticationFailed
  nameWithType: ErrorResponseCategory.AuthenticationFailed
  fullName: System.DirectoryServices.Protocols.ErrorResponseCategory.AuthenticationFailed
- uid: System.DirectoryServices.Protocols.ErrorResponseCategory
  parent: System.DirectoryServices.Protocols
  isExternal: false
  name: ErrorResponseCategory
  nameWithType: ErrorResponseCategory
  fullName: System.DirectoryServices.Protocols.ErrorResponseCategory
- uid: System.DirectoryServices.Protocols.ErrorResponseCategory.ConnectionClosed
  parent: System.DirectoryServices.Protocols.ErrorResponseCategory
  isExternal: false
  name: ConnectionClosed
  nameWithType: ErrorResponseCategory.ConnectionClosed
  fullName: System.DirectoryServices.Protocols.ErrorResponseCategory.ConnectionClosed
- uid: System.DirectoryServices.Protocols.ErrorResponseCategory.CouldNotConnect
  parent: System.DirectoryServices.Protocols.ErrorResponseCategory
  isExternal: false
  name: CouldNotConnect
  nameWithType: ErrorResponseCategory.CouldNotConnect
  fullName: System.DirectoryServices.Protocols.ErrorResponseCategory.CouldNotConnect
- uid: System.DirectoryServices.Protocols.ErrorResponseCategory.GatewayInternalError
  parent: System.DirectoryServices.Protocols.ErrorResponseCategory
  isExternal: false
  name: GatewayInternalError
  nameWithType: ErrorResponseCategory.GatewayInternalError
  fullName: System.DirectoryServices.Protocols.ErrorResponseCategory.GatewayInternalError
- uid: System.DirectoryServices.Protocols.ErrorResponseCategory.MalformedRequest
  parent: System.DirectoryServices.Protocols.ErrorResponseCategory
  isExternal: false
  name: MalformedRequest
  nameWithType: ErrorResponseCategory.MalformedRequest
  fullName: System.DirectoryServices.Protocols.ErrorResponseCategory.MalformedRequest
- uid: System.DirectoryServices.Protocols.ErrorResponseCategory.NotAttempted
  parent: System.DirectoryServices.Protocols.ErrorResponseCategory
  isExternal: false
  name: NotAttempted
  nameWithType: ErrorResponseCategory.NotAttempted
  fullName: System.DirectoryServices.Protocols.ErrorResponseCategory.NotAttempted
- uid: System.DirectoryServices.Protocols.ErrorResponseCategory.Other
  parent: System.DirectoryServices.Protocols.ErrorResponseCategory
  isExternal: false
  name: Other
  nameWithType: ErrorResponseCategory.Other
  fullName: System.DirectoryServices.Protocols.ErrorResponseCategory.Other
- uid: System.DirectoryServices.Protocols.ErrorResponseCategory.UnresolvableUri
  parent: System.DirectoryServices.Protocols.ErrorResponseCategory
  isExternal: false
  name: UnresolvableUri
  nameWithType: ErrorResponseCategory.UnresolvableUri
  fullName: System.DirectoryServices.Protocols.ErrorResponseCategory.UnresolvableUri
